README.md

```diff
@@ -8,7 +8,6 @@ This software is in very early alpha (use at your own risk). Only local sandboxe
 - MacOS (x86) or Linux (x86) for sandbox tests. Testnet is available regardless
 
 ## Include it in our project
-NOTE: since this is still alpha, we'll need to pull it down from github instead:
```

Comment:
thanks for noticing this one. I went to update the dependencies way back but somehow to get rid of the footnote